js/base.js

```diff
@@ -706,7 +706,7 @@ window.create_unified_changeset_notice = ({
     })
 }
 
-const createPnotifyAlert = (passedInConfig) => {
+const   createPnotifyAlert = (passedInConfig) => {
```

Comment:
what changed here?

Reply:
I'm betting that was actually a fat finger.   Fixed.